ddtrace/contrib/flask/patch.py

```diff
@@ -463,29 +463,36 @@ def _traced_request(pin, wrapped, instance, args, kwargs):
 
         This wrapper will add identifier tags to the current span from `flask.app.Flask.wsgi_app`.
         """
-        span = pin.tracer.current_span()
-        if not pin.enabled or not span:
+        current_span = pin.tracer.current_span()
```

Comment:
Can we revert this variable name change? Makes the PR much bigger than it needs to be